indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,324 @@
+                {% set should_render = (
+                    entry.type.name == 'SESSION_BLOCK' or
+                    (entry.type.name == 'CONTRIBUTION' and config.show_contribs) or
+                    (entry.type.name == 'BREAK' and config.show_breaks)
+                ) %}
+
+                {% if should_render %}
+                    <div class="day-entry">
+                        {% if entry.type.name != 'CONTRIBUTION' %}
+                            {{ _get_color_styles(entry) }}
+                        {% endif %}
+                        <div class="time-text time-from">
+                            {{ entry.start_dt|format_time(timezone=event.tzinfo) }}
+                        </div>
+                        <section class="{{ _get_custom_id(entry) }}">
+                            {{ _render_block_date(entry, day, config.print_date_close_to_sessions) }}
+                            <section class="session-block {{ 'contribution' if entry.type.name == 'CONTRIBUTION' else '' }}">
+                                <div class="session-block-wrapper">
+                                    {% if entry.type.name == 'BREAK' %}
+                                        {{ _render_break(entry, config, program_config) }}
+                                    {% elif entry.type.name == 'CONTRIBUTION' %}
+                                        {{
+                                            _render_contrib(
+                                                entry,
+                                                config,
+                                                program_config,
+                                                show_children_location
+                                            )
+                                        }}
+                                    {% else %}
+                                        {{
+                                            _render_session_block(
+                                                entry,
+                                                config,
+                                                program_config,
+                                                show_children_location
+                                            )
+                                        }}
+                                    {% endif %}
+                                </div>
+                            </section>
+                        </section>
+                        <div class="time-text time-to">
+                            {{ entry.end_dt|format_time(timezone=event.tzinfo) }}
+                        </div>
+                    </div>
+                {% endif %}
+            {% endfor %}
+        </div>
+    {% endfor %}
+{% endmacro %}
+
+{% macro _render_break(entry, config, program_config) %}
+    {% set break = entry.object %}
+
+    <section
+        id="entry-{{ entry.id }}"
+        class="session-block-title inherited-bg-color inherited-txt-color"
+    >
```

Comment:
Usually we wrap HTML like this:

```suggestion
    <section id="entry-{{ entry.id }}"
             class="session-block-title inherited-bg-color inherited-txt-color">
```

This is still something from before the time where we started using JSX and kind of got used to your style of wrapping. In any case, as long as it's consistent within a file I don't insist on changing it since the way you wrapped it does have its advantages (such as cleaner git diffs when changing attributes)

Reply:
I checked and in this file I've consistently used my original approach. Based on your comment (and my own opinion) I'd say let's indeed keep it like this.